docs/hub/collections.md

```diff
@@ -75,6 +75,23 @@ It's possible to add a note to any item in a collection to give it more context
 
 ![Collection note](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collection-note.webp)
 
+### Adding images to a collection item
+
+Similarily, you can attach images to a collection item. This is useful for showcasing the output of a model, the content of a dataset, attaching an infographic for context, etc.
+
+To start adding images to your collection, you can click on the image icon in the contextual menu of an item. The menu shows up when you hover over an item with your mouse.
+
+![Collection image icon](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/collections-image-button.webp)
```

Comment:
Please also add dark mode images

Reply:
I think keeping only images in light theme is on purpose cc @gary149

Reply:
In the rest of the docs we have images in dark mode as well

Reply:
I don't think so in most places, but yes, but yes it would be nicer.

Reply:
Opened https://github.com/huggingface/hub-docs/issues/1037